src/core/messenger/scroll.spec.ts

```diff
@@ -91,8 +90,8 @@ describe.skip('Cross-frame messenger: scroll', () => {
 			constructor(callback: ObsCallback) {
 				onIntersect = callback;
 				return Object.freeze({
-					observe: noop,
-					unobserve: noop,
+					observe: () => {},
```

Comment:
Does it mean here we can have a callback but before it was using function noop?

Reply:
I think this is okay as this file is a test. I think its just creating a mock `IntersectionObserver` and we're stubbing some of the interface we're not concerned with in context of the test.